draft-ldbc-cats-framework.md

```diff
@@ -230,11 +230,41 @@ CATS Service ID (CS-ID):
 CATS Instance Selector ID (CIS-ID):
 : An identifier of a specific service contact instance.
 
-## CATS Components {#sec-cat-arch}
+## CATS Framework {#sec-cats-framework}
+
+The high-level CATS framework can be illustrated in Figure 1.  Starting from the bottom part of the figure and moving towards the upper part, the following planes are defined.
+
+~~~                           
+   +----------------------------------+  |         +--------+
+   |         Management Plane         |  |  Intf   |        |
```

Comment:
Intf should be defined

Reply:
let's leave it to next commit or revision